snap/info.go

```diff
@@ -49,6 +49,9 @@ type PlaceInfo interface {
 	// SnapRevision returns the revision of the snap.
 	SnapRevision() Revision
 
+	// SnapRevisionFileName returns the name of the snap with the revision number, as stored on the filesystem.
+	SnapRevisionFileName() string
```

Comment:
we mostly use the spelling filename,  not file name.  Let's just call this Filename()